Common/Data/Market/FuturesContract.cs

```diff
@@ -0,0 +1,133 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using QuantConnect.Securities;
+using QuantConnect.Securities.Future;
+
+namespace QuantConnect.Data.Market
+{
+    /// <summary>
+    /// Defines a single futures contract at a specific expiration 
+    /// </summary>
+    public class FuturesContract
+    {
+        /// <summary>
+        /// Gets the futures contract's symbol
+        /// </summary>
+        public Symbol Symbol
+        {
+            get; private set;
+        }
+
+        /// <summary>
+        /// Gets the underlying security's symbol
+        /// </summary>
+        public Symbol UnderlyingSymbol
+        {
+            get; private set;
+        }
+
+
+        /// <summary>
+        /// Gets the expiration date
+        /// </summary>
+        public DateTime Expiry
+        {
+            get { return Symbol.ID.Date; }
+        }
+
+        /// <summary>
+        /// Gets the local date time this contract's data was last updated
+        /// </summary>
+        public DateTime Time
+        {
+            get; set;
+        }
+
+        /// <summary>
+        /// Gets the open interest
+        /// </summary>
+        public decimal OpenInterest
+        {
+            get; set;
+        }
+
+        /// <summary>
+        /// Gets the last price this contract traded at
+        /// </summary>
+        public decimal LastPrice
+        {
+            get; set;
+        }
+
+        /// <summary>
+        /// Gets the current bid price
+        /// </summary>
+        public decimal BidPrice
+        {
+            get; set;
+        }
+
+        /// <summary>
+        /// Get the current bid size
+        /// </summary>
+        public long BidSize
+        {
+            get; set;
+        }
+
+        /// <summary>
+        /// Gets the ask price
+        /// </summary>
+        public decimal AskPrice
+        {
+            get; set;
+        }
+
+        /// <summary>
+        /// Gets the current ask size
+        /// </summary>
+        public long AskSize
+        {
+            get; set;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FuturesContract"/> class
+        /// </summary>
+        /// <param name="symbol">The futures contract symbol</param>
+        /// <param name="underlyingSymbol">The symbol of the underlying security</param>
+        public FuturesContract(Symbol symbol, Symbol underlyingSymbol)
+        {
+            Symbol = symbol;
+            UnderlyingSymbol = underlyingSymbol;
+        }
+
+        /// <summary>
+        /// Returns a string that represents the current object.
+        /// </summary>
+        /// <returns>
+        /// A string that represents the current object.
+        /// </returns>
+        public override string ToString()
+        {
+            return Symbol.Value;
```

Comment:
Yes, I grouped options/futures symbol representation into SymbolRepr.cs file. I still sometimes see SIDs, and will have to clean it up later.